src/components/CoinItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const CoinItem = (props) => {
+  const { coin } = props;
+
+  return (
+    <li className={`singleCoin ${coin.checker}`}>
+      <Link to={`/coin/${coin.id}`} className="nav-link">
+        <p>→</p>
+      </Link>
+      <div>
+        <p>{coin.id}</p>
+        <h3>{coin.name}</h3>
+        <p>{coin.symbol}</p>
+      </div>
+    </li>
+  );
+};
+
+CoinItem.propTypes = {
+  coin: PropTypes.instanceOf(Array).isRequired,
+};
```

Comment:
- [ ] Kindly fix this error which appears in the browser console as a result of your passing in a prop of an unexpected type. Did you mean to have this as an Array instead of an Object? See error below 👇 
![image](https://user-images.githubusercontent.com/46839548/182955341-f0b38310-d388-4784-bea4-29cac4cfa904.png)